<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterRunner</class>
 <widget class="QWidget" name="MasterRunner">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MasterRunner</string>
  </property>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="animal_input">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Animal ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="program">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Imager</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Miniscpe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neuropixels</string>
    </property>
   </item>
  </widget>
  <widget class="QLCDNumber" name="frames">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="program_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Program</string>
   </property>
  </widget>
  <widget class="QLedLabel" name="led">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLedLabel</class>
   <extends>QLabel</extends>
   <header>qledlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
